src/add-ons/kernel/drivers/nvmm/nvmm_ioctl.h

```diff
@@ -144,6 +150,28 @@ struct nvmm_ioc_ctl {
 	size_t size;
 };
 
+#if defined(__HAIKU__)
+// These ioctl opcodes are not unique in Haiku
```

Comment:
To be honest, we could probably just define a custom `_IOR` instead of doing this manually.